### PR TITLE
NN III

### PR DESCRIPTION
**Pull request by trgardos**
Reformatted for Quarto and for slides.

Addresses #173

### CODE REVIEW

**Comment by trgardos** [comment]
ok. @sladenheim  -- ready for review

File: ds701_book/25-NN-III-CNNs.qmd [hunk 1]
```diff
@@ -372,124 +377,121 @@ Problems with fully-connected networks
 * Should be stable under transformations
```
Comment (sladenheim) [inline ds701_book/25-NN-III-CNNs.qmd:377]: This sentence does not make sense to me. How is it a problem? If they should be stable under transformations wouldn't that be a good thing?

File: ds701_book/25-NN-III-CNNs.qmd [hunk 2]
```diff
@@ -372,124 +377,121 @@ Problems with fully-connected networks
...
     * Don’t want to re-learn appearance at different parts of image
 
+## Classification Invariant to Shift
+
 <!-- Image Credit "https://udlbook.github.io/udlbook/"-->
 
-#### Classification Invariant to Shift
+![](figs/NN-figs/L25-shift-img-class.png){width="75%" fig-align="center"}
```
Comment (sladenheim) [inline ds701_book/25-NN-III-CNNs.qmd:384]: Can you add a sentence to explain what this figure is intended to convey?

File: ds701_book/25-NN-III-CNNs.qmd (hunk 1 above)
Comment (trgardos) [reply ds701_book/25-NN-III-CNNs.qmd:429]: that was referring to FCNs. The slide title was incorrect. I changed it and moved it to the beginning of the section in the next commit.

File: ds701_book/25-NN-III-CNNs.qmd (hunk 2 above)
Comment (trgardos) [reply ds701_book/25-NN-III-CNNs.qmd:436]: done in next commit.

**Comment by trgardos** [comment]
@sladenheim  -- addressed your review comments in last commit

**Review by sladenheim — APPROVED** [review]

**Pull request merged**